Gemfile.lock

```diff
@@ -193,7 +193,7 @@ GEM
       thread_safe (~> 0.3, >= 0.3.1)
     barby (0.6.8)
     base32-crockford (0.1.0)
-    base64 (0.1.1)
+    base64 (0.2.0)
```

Comment:
same comment as here, I'm wary of upgrading base64 at this time

https://github.com/18F/identity-idp/pull/9644#discussion_r1402410988

Reply:
Removed the base64 update from this PR